flocker/control/_model.py

```diff
@@ -199,6 +201,7 @@ class Application(PRecord):
     restart_policy = field(mandatory=True, initial=RestartNever())
     environment = field(mandatory=True, initial=pmap(), factory=pmap,
                         type=PMap)
+    running = field(mandatory=True, initial=True, type=bool)
 
```

Comment:
Docker containers have a  bunch of states they can be in apart from just "running" and "not running".  Maybe addressing that is out of scope for this issue - it's not like we were representing any of those other states prior to this change, after all.

If I were building an application on top of our container API I'd probably want to see lots more information about what `docker inspect` calls `State`.  For example, the way restart policies are reflected is running=false, restarting=true.  I'd probably want to do something differently in that case and just having running=false doesn't tell me enough.

Yea, the more I write here the more this seems out of scope.

What is the use-case for having a default here?  I'm guessing "convenience in maintaining the test suite"?